kolibri/plugins/coach/assets/src/views/reports/ExerciseQuestionListPageBase.vue

```diff
@@ -81,11 +81,12 @@
     mixins: [commonCoach],
     computed: {
       ...mapState('questionList', ['exercise']),
+      ...mapState('resourceDetail', ['resource']),
       ...mapGetters('questionList', ['difficultQuestions']),
       lesson() {
         return this.lessonMap[this.$route.params.lessonId];
       },
-      resource() {
+      exercise() {
```

Comment:
I think this computed prop can just be deleted, as both exercise and resource are coming from the vuex state now.